rfc/0054-sdk3-management-apis.md

```diff
@@ -2977,6 +2987,14 @@ enum BucketType {
 
 * `MinimumDurabilityLevel` (`DurabilityLevel`) - The minimum durability level to use for all KV operations.
 
+* `StorageBackend` ([`StorageBackend`](#storagebackend)) - The storage type to use (note: Magma is EE only).
+
+* `StorageQuotaPercentage` (`int`) - The storage quota trigger threshold as a percentage (only supported with magma).
+
+* `MagmaFragmentationPercentage` (`int`) - The fragmentation trigger threshold as a percentage (only supported with magma).
```

Comment:
I'm not 100% sure that this is the field intended but this field works.

Reply:
Also, per latest discussion at team meeting on 12/01/2021 we are only adding the `magma` StorageBackend. I've updated CBD-4373 and these percentages need to be removed here as well.